src/sidebar/components/tab.vue

```diff
@@ -187,6 +187,33 @@ function onMouseDown(e: MouseEvent): void {
   else if (e.button === 1) {
     e.preventDefault()
     Mouse.blockWheel()
+
+    if (e.ctrlKey) {
```

Comment:
This isn't really following the same style as the existing code, where instead of returning, it makes sure that everything is an else if.

That would mean removing the return statements below, and then the existing lines 202 - 204 would become } else if ... {

Same deal on existing lines 215-217.

Reply:
Fair point, I'll update it after merge.

Reply:
Hm, no, not fair :), it's simpler to use `return` statement to get the 'fallback' logic (with 'none' values)

Reply:
Yes, fallback to 'none' is exactly why I made it in `return` style. Some people may rely on Ctrl+Middle click to behave just as regular middle click (for whatever reason). 

I was also thinking about writing it in a bit more complex way, so that actions like "duplicate" and "discard" would work with middle click [drag selection](https://github.com/mbnuqw/sidebery/pull/754)— but this way it wouldn't work with "Edit title" action (and I really wanted to have it there), so I guess maybe next time. It's going to be too many checks to handle all possible actions this way.